tests/system/python/e2e/test_south_service_tuning.py

```diff
@@ -0,0 +1,371 @@
+# -*- coding: utf-8 -*-
+
+# FLEDGE_BEGIN
+# See: http://fledge-iot.readthedocs.io/
+# FLEDGE_END
+
+""" Test south service tuning parameters for bufferThreshold and maxSendLatency """
+
+import subprocess
+import http.client
+import json
+import time
+import urllib.parse
+import pytest
+
+from helpers import utils
+
+__author__ = "Devki Nandan Ghildiyal"
+__copyright__ = "Copyright (c) 2025 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+SERVICE_NAME = "TuningSouth"
+#PUT_URL = f"/fledge/category/{SERVICE_NAME}Advanced"
+
+#clean_setup_fledge_packages, 
+class TestSouthServiceTuning:
+    
+    def test_south_service_tuning_buffer_threshold(self, reset_and_start_fledge, fledge_url, 
+                                                   wait_time, retries, add_south, south_branch, plugin_language, enable_schedule, disable_schedule, plugin_name):
+        """ Test south service tuning parameters - bufferThreshold and maxSendLatency
+            
+            This test:
+            1. Sets up a south service using sinusoid plugin
+            2. Configures polling interval to 2 seconds (30 readingsPerSec)
+            3. Sets maxSendLatency to 60000ms (1 minute)
+            4. Sets bufferThreshold to 200
+            5. Verifies 30 readings after 90 seconds (1 minute worth)
+            6. Tests dynamic parameter changes
+        """
+        
+        print("\n=== Running South Service Tuning Test ===")
+
+        # Step 1: Create south service with sinusoid plugin and initial configuration
+        # Create the service
+        response = add_south(plugin_name, south_branch, fledge_url, service_name=SERVICE_NAME, plugin_lang=plugin_language, start_service=False)
+        assert SERVICE_NAME == response["name"]
+        print(f"Created south service: {SERVICE_NAME}")
+        time.sleep(2)  # Allow time for Advance category creation
+        
+        # Configure advanced parameters after service creation
+        advanced_config = {
+            "units": "minute" ,            # Polling interval unit
+            "readingsPerSec": "30",        # 2 seconds interval
+            "maxSendLatency": "60000",     # 1 minute
+            "bufferThreshold": "200"       # Buffer 200 readings
+        }
+        put_url = f"/fledge/category/{SERVICE_NAME}Advanced"
+        print(f"Configuring advanced parameters for {SERVICE_NAME}: {advanced_config}", put_url)
+        resp = utils.put_request(fledge_url, urllib.parse.quote(put_url), advanced_config)
```

Comment:
Above code is repeated multiple times in different test cases, we should create a common function to remove redundancies